sauna/plugins/ext/http.py

```diff
@@ -0,0 +1,60 @@
+from sauna.plugins import Plugin, PluginRegister
+
+my_plugin = PluginRegister('HTTP')
+
+
+@my_plugin.plugin()
+class HTTP(Plugin):
+
+    def __init__(self, config):
+        super().__init__(config)
+        try:
+            import requests
+            self.requests = requests
+        except ImportError:
+            from ... import DependencyError
+            raise DependencyError(self.__class__.__name__, 'requests',
+                                  'requests', 'python3-requests')
+
+    @my_plugin.check()
+    def request(self, check_config):
+        method = check_config.get('method', 'GET').upper()
+        timeout = check_config.get('timeout', 10)
+        code = check_config.get('code', 200)
+        content = check_config.get('content', '')
+
```

Comment:
Monitoring time could be really great too.
Can we add a config to set this ?
like : 
- warning_response_time
- critical_response_time